### PR TITLE
Removed _connectivity_channel tests

### PR DESCRIPTION
**Pull request by kpayson64**
A quick grep for "_connectivity_channel" shows that this module is only used in the test, and is not used by either the GA or Beta API.

### CODE REVIEW

File: src/python/grpcio/tests/unit/beta/_connectivity_channel_test.py [hunk 1]
```diff
@@ -1,201 +0,0 @@
...
-        lambda connectivities:
-        connectivities[-1] is not interfaces.ChannelConnectivity.READY)
-    connectivity_channel.unsubscribe(callback.update)
-
-    server.shutdown()
-    server_completion_queue.shutdown()
-    server_completion_queue_thread.join()
-
-
-class ConnectivityStatesTest(unittest.TestCase):
```
Comment (nathanielmanistaatgoogle) [inline src/python/grpcio/tests/unit/beta/_connectivity_channel_test.py]: Please retain this test until the beta API is removed (six to twelve months from now?). Feel free to move it to another module if you like?

**Comment by kpayson64** [comment]
I've re-added the ConnectivityStatesTest.

**Comment by nathanielmanistaatgoogle** [comment]
Change content looks great but please correct the commit log message?

**Comment by kpayson64** [comment]
Updated commit message.

**Comment by nathanielmanistaatgoogle** [comment]
Luhguhtum, but please consider dropping the " test" from "_connectivity_channel.ChannelConnectivityTest test". :-P

**Pull request merged**